situp-plugins/apmtracesource/src/main/java/com/amazon/situp/plugins/processor/oteltrace/OTelTraceRawProcessor.java

```diff
@@ -0,0 +1,145 @@
+package com.amazon.situp.plugins.processor.oteltrace;
+
+import com.amazon.situp.model.processor.Processor;
+import com.amazon.situp.model.record.Record;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.protobuf.util.JsonFormat;
+import io.opentelemetry.proto.trace.v1.ResourceSpans;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.time.Instant;
+import java.util.*;
+
```

Comment:
Currently, we still need to annotate the plugin with `@SitupPlugin(name = "otel_trace_processor", type = PluginType.PROCESSOR)` for it to be available at Runtime

Note: _u can select a different name if you would like_

Reply:
Looks like I deleted it by mistake, will include it